docs/features/openapi.securitydefinitions.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Adding security information to OpenAPI documentation"
+layout: default
+---
+
+## Adding OAuth security definition to API operations
+
+### Introduction
+
+When an API is secured via OAuth, it is helpful if the Open API documentation makes this clear via a security scheme and the API operations that require authorization automatically inform the consumer that it is possible that a 401 (Unauthorized) or 409 (Forbidden) response is returned.
+The `OAuthAuthorizeOperationFilter` that is part of this package exposes this functionality.
+
+### Usage
+
+To indicate that an API is protected by OAuth, and enable Swagger UI to retrieve a JWT token when invoking API operations via Swagger, add the following code to the `ConfigureServices` method of the `Startup` class:
+
+```csharp
+services.AddSwaggerGen(c =>
+{
+   c.SwaggerDoc("v1", new Info { Title = "My API v1", Version = "v1" });
+
+   c.AddSecurityDefinition("oauth2", new OAuth2Scheme
+   {
+      Flow = "implicit",
+      AuthorizationUrl = $"{authorityUrl}connect/authorize",
+      Scopes = scopes
+   });
+
+   c.OperationFilter<AuthorizeCheckOperationFilter>("myApiScope1", "myApiScope2");
```

Comment:
I suspect that you should use our operation filter here instead of the `AuthorizeCheckOperationFilter` from a [demo project](https://github.com/dvanherten/swagger-identityserver/blob/master/WebApi/AuthorizeCheckOperationFilter.cs). :)

Reply:
What do you mean with "our operation filter"? Are you referring to the shared access signature + certificate filter?

If so, we've discussed this and will exclude it in the first iteration.

Reply:
The operation filter in the PR :)

Reply:
Ah I see, naming is not inline, sorry about that :)

Reply:
Should be fixed.

Reply:
Thanks!